frontend/www/js/omegaup/arena/contest_practice.ts

```diff
@@ -112,11 +117,30 @@ OmegaUp.on('ready', () => {
   // The hash is of the form `#problems/${alias}`.
   const problemMatch = /#problems\/([^/]+)/.exec(window.location.hash);
   const problemAlias = problemMatch?.[1] ?? null;
-  if (problemAlias) {
+  if (problemAlias && problemMatch) {
     // This needs to be set here and not at the top because it depends
     // on the `navigate-to-problem` callback being invoked, and that is
     // not the case if this is set a priori.
     contestPractice.problem = { alias: problemAlias, runs: [] };
+    const popupDisplayed = problemMatch.input.split('/')[2];
+    if (popupDisplayed) {
+      if (popupDisplayed === 'new-run') {
+        contestPractice.popupDisplayed = PopupDisplayed.RunSubmit;
+      }
+      if (popupDisplayed.includes('show-run')) {
```

Comment:
```suggestion
      } else if (popupDisplayed?.includes('show-run')) {
```

así L126 se puede ir.

Reply:
Por suerte esto ya no fue necesario gracias al comentario de las capturas nombradas y grupos opcionales